src/components/navbars/NavbarSide.vue

```diff
@@ -0,0 +1,51 @@
+<script lang="ts" setup>
+import { Icon } from '@iconify/vue';
+import { useI18n } from 'vue-i18n';
+
+import messages from "../../i18n/index"
+
+const i18n = useI18n({
+  messages,
+  useScope: "local"
+})  
+
+const {t} = i18n;
+
+interface NavLink {
+  tKey: string,
+  to: string,
+  icon: string
+}
+
+const links: NavLink[] = [
+  {tKey: "navbar.tricks", to: "/tricks", icon: "ic:baseline-auto-awesome"},
+  {tKey: "navbar.combos", to:"/combos", icon: "ic:baseline-spoke"},
+  {tKey: "navbar.glossary", to: "/glossary", icon: "ic:sharp-menu-book"},
+  {tKey: "navbar.generator", to: "/generator", icon: "ic:sharp-factory"},
+  {tKey: "navbar.more", to:"/more", icon: "ic:baseline-more-horiz"}
```

Comment:
`/about` would be more fitting

Reply:
That is what I called it at first. Then I changed it, as there might be settings under that page as well. Though I think you are still correct and at least the route should be called `/about`. I'll still leave the name of the button ("More") the same though.

Reply:
Changed route from `/more` to `/about`, left button name the same ("More").